swatch-common-export/src/main/java/com/redhat/swatch/export/ExportRequestHandler.java

```diff
@@ -18,82 +18,43 @@
  * granted to use or replicate Red Hat trademarks that are incorporated
  * in this software or its documentation.
  */
-package org.candlepin.subscriptions.export;
-
-import static org.candlepin.subscriptions.export.ExportSubscriptionConfiguration.SUBSCRIPTION_EXPORT_QUALIFIER;
+package com.redhat.swatch.export;
 
 import com.redhat.cloud.event.parser.ConsoleCloudEventParser;
-import com.redhat.swatch.clients.export.api.client.ApiException;
-import com.redhat.swatch.clients.export.api.model.DownloadExportErrorRequest;
-import com.redhat.swatch.clients.export.api.resources.ExportApi;
-import io.micrometer.core.annotation.Timed;
-import jakarta.ws.rs.core.Response;
-import jakarta.ws.rs.core.Response.Status;
+import com.redhat.swatch.export.api.ExportDelegate;
+import com.redhat.swatch.export.api.RbacDelegate;
 import java.io.File;
 import java.io.IOException;
 import java.util.List;
 import java.util.stream.Stream;
+import lombok.AllArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
-import org.candlepin.subscriptions.exception.ExportServiceException;
-import org.candlepin.subscriptions.rbac.RbacApiException;
-import org.candlepin.subscriptions.rbac.RbacService;
-import org.candlepin.subscriptions.task.TaskQueueProperties;
-import org.candlepin.subscriptions.util.KafkaConsumerRegistry;
-import org.candlepin.subscriptions.util.SeekableKafkaConsumer;
-import org.springframework.beans.factory.annotation.Qualifier;
-import org.springframework.kafka.annotation.KafkaListener;
-import org.springframework.stereotype.Service;
-import org.springframework.transaction.annotation.Transactional;
 
-/** Listener for Export messages from Kafka */
-@Service
 @Slf4j
-public class ExportSubscriptionListener extends SeekableKafkaConsumer {
+@AllArgsConstructor
+public class ExportRequestHandler {
 
   public static final String ADMIN_ROLE = ":*:*";
   public static final String SWATCH_APP = "subscriptions";
   public static final String MISSING_PERMISSIONS = "Insufficient permission";
+  public static final int INTERNAL_ERROR = 500;
+  public static final int FORBIDDEN_STATUS = 403;
```

Comment:
How do you feel about dropping these and just using `HttpURLConnection.HTTP_FORBIDDEN`, etc?

Reply:
+1